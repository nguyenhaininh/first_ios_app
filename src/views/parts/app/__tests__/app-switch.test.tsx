/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import AppSwitch from '../app-switch';
import { View } from 'react-native';

import * as renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';

it('renders correctly', () => {
  const onSwitchTab = jest.fn();
  const tree = renderer.create(
    <MemoryRouter>
      <AppSwitch>
        <View key="a" />
        <View key="b" />
        <View key="c" />
      </AppSwitch>
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
