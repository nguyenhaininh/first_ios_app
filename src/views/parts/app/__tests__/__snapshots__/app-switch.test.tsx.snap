// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    pointerEvents="none"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      isActive={false}
    />
  </View>
  <View
    pointerEvents="none"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      isActive={false}
    />
  </View>
  <View
    pointerEvents="none"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      isActive={false}
    />
  </View>
</View>
`;
